File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/Table_of_Contents/Merge_Table_of_Contents_Description.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF.Table_of_Contents
{
    public partial class Merge_Table_of_Contents_Description : UserControl
    {
        public Merge_Table_of_Contents_Description()
        {
            InitializeComponent();
        }

        private void Merge_Table_of_Contents_Description_Load(object sender, EventArgs e)
        {
            descriptionWebBrowser.Visible = true;
            string descriptionString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\Table_of_Contents\Multiple_HTML.html");
            descriptionWebBrowser.DocumentText = descriptionString;
        }
    }
}

File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/Table_of_Contents/Merge_Table_of_Contents_Description.vb
Public Class Merge_Table_of_Contents_Description

    Private Sub Merge_Table_of_Contents_Description_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        descriptionWebBrowser.Visible = True
        Dim descriptionString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\Table_of_Contents\Multiple_HTML.html")
        descriptionWebBrowser.DocumentText = descriptionString
    End Sub
End Class
